From 3b9caa0ba59d0b259d39a0aef47ccb72218d3d6a Mon Sep 17 00:00:00 2001
From: Jenkins Slave <jenkins@code.rdkcentral.com>
Date: Wed, 8 Dec 2021 11:20:12 +0000
Subject: [PATCH] 2062-Fix-RDK-log-build-failure

---
 src/pl2rld/pl2rld.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pl2rld/pl2rld.c b/src/pl2rld/pl2rld.c
index 842b78f..5615818 100644
--- a/src/pl2rld/pl2rld.c
+++ b/src/pl2rld/pl2rld.c
@@ -338,7 +338,7 @@ void pl2rld_client_recv_log(pclient_t *pc, pl2rl_msg_t *msg, char *text)
 
     (void)sev; // currently not used
     RDK_LOG(rdk_level, RDK_LOGGER_MODULE,
-            "%s[%ld]: %s: %.*s\n",
+            "%s[%u]: %s: %.*s\n",
             pc->name,
             pc->pid,
             mod,
-- 
2.34.1

